e2e/test_operator/operator_test.go

```diff
@@ -113,6 +113,9 @@ var _ = BeforeSuite(func() {
 		factory.GetClusterOptions()...,
 	)
 
+	// Load some data async into the cluster. We will only block as long as the Job is created.
+	factory.CreateDataLoaderIfAbsent(fdbCluster)
```

Comment:
I assume the loading of data will happen in parallel with the tests running. Depending on how long it will take to load data, a subset of tests could run with data already loaded? Do you think this will introduce non-determinism, and we might not see the issues (that we see in nightly runs) when we run individual tests standalone?

Reply:
I can change this to wait that the Job has completed, this obviously will slow down the tests and we can add another issue to add some load during the tests.